edgelet/support-bundle/src/shell_util.rs

```diff
@@ -0,0 +1,251 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use chrono::{DateTime, NaiveDateTime, Utc};
+use std::io::{Seek, Write};
+use tokio::process::Command;
+
+use failure::Fail;
+use zip::{write::FileOptions, ZipWriter};
+
+use crate::error::{Error, ErrorKind};
+use edgelet_core::LogOptions;
+
+pub async fn write_check(
+    writer: &mut impl Write,
+    iothub_hostname: Option<String>,
+    verbose: bool,
+) -> Result<(), Error> {
+    print_verbose("Calling iotedge check", verbose);
+
+    let mut iotedge = std::env::current_exe().unwrap();
+    if iotedge.to_string_lossy().contains("aziot-edged") {
+        print_verbose(
+            "Calling iotedge check from edgelet, using iotedge from path",
+            verbose,
+        );
```

Comment:
Does it make sense to consolidate these prints into 1 that is something like this?
```rust
print_verbose("Calling iotedge check with iotedge {}", iotedge.to_string_lossy(), verbose);
```

Or do we only really care about what things are named now in 1.2.X?

Reply:
print_verbose is just a local function I made, not a macro. It could be a macro, but I was just keeping it simple